zerver/views/unsubscribe.py

```diff
@@ -18,7 +18,9 @@ def process_unsubscribe(
     unsubscribe_function: Callable[[UserProfile], None],
 ) -> HttpResponse:
     try:
-        user_profile = get_object_from_key(confirmation_key, [Confirmation.UNSUBSCRIBE])
+        user_profile = get_object_from_key(
+            confirmation_key, [Confirmation.UNSUBSCRIBE], mark_object_used=False
+        )
```

Comment:
This is a bit similar in that  the object on the Confirmation doesn't have a `status` - but given the nature of an "unsubscribe" link, the possibility of re-use is not bad. So not sure if we want to change this or not.

Reply:
I think it's probably fine to leave as-is, but good to note.